.bemlintrc.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Please do not add useless files to your PR.
You can use **git add src/** command and **git add readme.md** command to add needed files
**git status** command — for checking file status